SpywareFilter/sections/general_extensions.txt

```diff
@@ -20,6 +20,8 @@ imdb.com#%#//scriptlet("set-constant", "doWithAds", "noopFunc")
 ! https://developer.chrome.com/docs/privacy-sandbox/topics/
 ! TODO: replace with a scriptlet later: https://github.com/AdguardTeam/Scriptlets/issues/214
 #%#//scriptlet('set-constant', 'Document.prototype.browsingTopics', 'undefined')
+! https://github.com/AdguardTeam/AdguardFilters/pull/134619
+auction.co.kr#%#AG_onLoad(() => { Array.from(document.querySelectorAll('a[href*="through.auction.co.kr/common/SafeRedirect.aspx"]')).forEach((e) => { e.href = new URLSearchParams(e.href).get('next') }) })
```

Comment:
What the purpose of this rule?

Reply:
What do they collect here? I see url even without redundant parameters.

Reply:
Hmm...Okay. I will delete that.